pkg/gui/controllers/files_controller.go

```diff
@@ -302,49 +307,61 @@ func (self *FilesController) stageAll() error {
 	return self.contexts.Files.HandleFocus()
 }
 
-func (self *FilesController) ignore(node *filetree.FileNode) error {
-	if node.GetPath() == ".gitignore" {
-		return self.c.ErrorMsg("Cannot ignore .gitignore")
-	}
+func (self *FilesController) unstageFiles(node *filetree.FileNode) error {
 
-	unstageFiles := func() error {
-		return node.ForEachFile(func(file *models.File) error {
-			if file.HasStagedChanges {
-				if err := self.git.WorkingTree.UnStageFile(file.Names(), file.Tracked); err != nil {
-					return err
-				}
+	return node.ForEachFile(func(file *models.File) error {
+		if file.HasStagedChanges {
+			if err := self.git.WorkingTree.UnStageFile(file.Names(), file.Tracked); err != nil {
+				return err
 			}
+		}
 
-			return nil
-		})
-	}
+		return nil
+	})
+}
+
+func (self *FilesController) checkTracking(node *filetree.FileNode, trText string, trPrompt string, trAction string, f func(string) error) error {
 
 	if node.GetIsTracked() {
 		return self.c.Confirm(types.ConfirmOpts{
-			Title:  self.c.Tr.IgnoreTracked,
-			Prompt: self.c.Tr.IgnoreTrackedPrompt,
+			Title:  trText,
+			Prompt: trPrompt,
 			HandleConfirm: func() error {
-				self.c.LogAction(self.c.Tr.Actions.IgnoreFile)
+				self.c.LogAction(trAction)
 				// not 100% sure if this is necessary but I'll assume it is
-				if err := unstageFiles(); err != nil {
+				if err := self.unstageFiles(node); err != nil {
 					return err
 				}
 
 				if err := self.git.WorkingTree.RemoveTrackedFiles(node.GetPath()); err != nil {
 					return err
 				}
 
-				if err := self.git.WorkingTree.Ignore(node.GetPath()); err != nil {
+				if err := f(node.GetPath()); err != nil {
 					return err
 				}
+
 				return self.c.Refresh(types.RefreshOptions{Scope: []types.RefreshableView{types.FILES}})
 			},
 		})
 	}
+	return nil
+}
+
+func (self *FilesController) ignore(node *filetree.FileNode) error {
+	if node.GetPath() == ".gitignore" {
+		return self.c.ErrorMsg("Cannot ignore .gitignore")
+	}
+
+	trakckingErr := self.checkTracking(node, self.c.Tr.IgnoreTracked, self.c.Tr.IgnoreTrackedPrompt, self.c.Tr.Actions.IgnoreFile, self.git.WorkingTree.Ignore)
```

Comment:
All occurrences of `trakcking` should be replaced with `tracking`

Reply:
@mark2185 done, sorry about that. I'm not the best a spelling 😅

Reply:
Nothing to be sorry about, that's what the reviews are for! :)

Reply:
@mark2185 so what next? dose it get merged now or before the next release?

Reply:
No idea, it's up to @jesseduffield to decide. 

I'm just a passerby eager to help.